Cargo.toml

```diff
@@ -23,6 +25,7 @@ futures-core = { version = "0.3", default-features = false }
 http = "1"
 http-body = "1"
 http-body-util = "0.1"
+httpdate = "1"
```

Comment:
I'm probably going to mark this as optional even though it's already a dependency of hyper. The change will do nothing other than get me in the habit of making sure that dependencies that are only used for a feature flag are specified the way that you would expect in Cargo.toml.